### ISSUE TITLE
fix regression over injection of extensions declared under `config.injectFileTypes`

### ISSUE THREAD

**Issue by zanona** [BrowserSync/browser-sync#1488]
Apparently, with the resolution of #10 we had a regression preventing file extensions declared under `config.injectFileTypes: ['css', 'less']` to be respected once updated where a hard reload is performed instead and only applied correctly for `.css` files

https://github.com/BrowserSync/browser-sync/blob/cb5b44c0e7393b6f083b1000848bba8fce602d9f/client/vendor/Reloader.ts#L64-L68

**Comment by shakyShane** [comment]
thanks @zanona - this will be addressed shortly

**Comment by shakyShane** [comment]
@zanona can you give browser-sync@2.23.4 a try and let me know how it goes :)

**Comment by zanona** [comment]
Hey @shakyShane, thanks for sorting that out.
There's just one pet peeve of mine that this changes brings as opposed to something I did locally to quickly path the issue which what simply add the less extension to css like:

```js
if (options.liveCSS) { 
     if (path.match(/\.(css|less)$/i)) { 
         if (this.reloadStylesheet(path)) { return; } 
     } 
 } 
```

Which is that the current fix on 2.23.4 flashes the page (as if it was removing all the styles and re-applying) every time a less file is saved but with the workaround above that transition is very smooth where you see updates only when saving without any FOUC.

If you compare between 2.23.4 and the quick fix above you will probably be able to see that :)

I hope this helps. All the best
